build: update to latest docs tooling to resolve missing answers

crisbeto opened the pull request:
Updates to the latest version of `@angular/docs` which should include a fix for the missing answers in the tutorials.

alxhub reviewed (approved):
Reviewed-for: global-approvers

crisbeto committed:
build: update to latest docs tooling to resolve missing answers

Updates to the latest version of `@angular/docs` which should include a fix for the missing answers in the tutorials.

thePunderWoman commented:
This PR was merged into the repository by commit ceaaa9815b0e16f07806a5d8de38e6a8bcdc8271.

The changes were merged into the following branches: main, 18.0.x
